src/test/java/com/sofrelgroup14/ci/CiApplicationTests.java

```diff
@@ -1,13 +1,21 @@
 package com.sofrelgroup14.ci;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.sofrelgroup14.ci.controller.BuildController;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
 class CiApplicationTests {
 
+	@Autowired
+	private BuildController controller;
+
 	@Test
 	void contextLoads() {
+		assertThat(controller).isNotNull(); // Source: https://spring.io/guides/gs/testing-web/
+		assertThat(true).isFalse();
```

Comment:
We definitely need more tests here. I found some weird looking (so called "smoke test") that seems to verify that the controller is up and running (see the url in the comment) so I used that to have at least one associated test in the feature commit (grading criteria P4). But yeah, more need to be added